### PR TITLE
Add support for jEnv in textconv-javap

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

**Comment by dwijnand** [comment]
Hey @retronym, any chance this can be accepted?

I've found using jEnv (and it's flexible configuration) to be superior to trying to micromanage JAVA_HOME (or JDK_HOME).

File: git-java/textconv-javap [hunk 1]
```diff
@@ -6,6 +6,7 @@ javapBin=javap
 [[ -x "$javapBin" ]] || javapBin="$JAVA_HOME/bin/javap"
 [[ -x "$javapBin" ]] || javapBin="/etc/alternatives/java_sdk/bin/javap"
 [[ -x "$javapBin" ]] || javapBin="/etc/alternatives/javap"
+[[ -x "$javapBin" ]] || javapBin="$HOME/.jenv/shims/javap"
```
Comment (retronym) [inline git-java/textconv-javap]: Is the leading `+` intentional here?

Comment (dwijnand) [inline git-java/textconv-javap]: uhh.. that's just the diff format, click "View" and you'll see what it looks like.

Comment (retronym) [inline git-java/textconv-javap]: Oh, yeah. Sorry, its my first day :smile:

Comment (dwijnand) [inline git-java/textconv-javap]: hehehe

**Pull request merged**